docs/languages/python.md

```diff
@@ -0,0 +1,83 @@
+---
+Order: 8
+Area: languages
+TOCTitle: Python
+ContentId: c2cb770d-571d-4edf-9eb9-b5b8977c21a0
+PageTitle: Python with Visual Studio Code
+DateApproved: 6/2/2016
+MetaDescription: Learn about Visual Studio Code features (syntax highlighting, snippets, linting) for Python development.
```

Comment:
I think we need to add the word "editor" to the MetaDescription for SEO. 
Something like "Learn about Visual Studio Code editor features (IntelliSense, debugging, linting and snippets) for Python."
I think we should update the other language topic metadata to include "editor" as well.